portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,9 @@
+::root{
+    font-family:"Roboto";
+    font-style:normal;
+    font-size:5vw;
+    --primary:#3D67D1;
```

Comment:
use only lowercase for the color literals, per style guide: https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
Thanks Krasimir, I've pushed a commit turning them to lowercase now.